examples/vanilla-ts-esm/public/mux-player.html

```diff
@@ -67,6 +67,7 @@
     <div class="buttons">
       <button class="dialog-show-btn">Show error dialog</button>
       <button class="dialog-close-btn">Close dialog</button>
+      <button class="switch-theme-btn">Switch Theme</button>
```

Comment:
blocking: since this current codebase is likely an "intermediary step" with how themes will work, I recommend not merging this example until we're confident in what we want our public-facing API to look like.

Reply:
good call! I removed this until we have this functioning properly